interchaintest/feegrant_test.go

```diff
@@ -93,429 +93,447 @@ func TestRelayerFeeGrant(t *testing.T) {
-	rep := testreporter.NewNopReporter()
-	eRep := rep.RelayerExecReporter(t)
-
-	client, network := interchaintest.DockerSetup(t)
-
-	// Build interchain
-	require.NoError(t, ic.Build(ctx, eRep, interchaintest.InterchainBuildOptions{
-		TestName:  t.Name(),
-		Client:    client,
-		NetworkID: network,
-
-		SkipPathCreation: false,
-	}))
-
-	t.Parallel()
-
-	// Get Channel ID
-	gaiaChans, err := r.GetChannels(ctx, eRep, gaia.Config().ChainID)
-	require.NoError(t, err)
-	gaiaChannel := gaiaChans[0]
-	osmosisChannel := gaiaChans[0].Counterparty
-
-	// Create and Fund User Wallets
-	fundAmount := int64(10_000_000)
-
-	// Tiny amount of funding, not enough to pay for a single TX fee (the GRANTER should be paying the fee)
-	granteeFundAmount := int64(10)
-	granteeKeyPrefix := "grantee1"
-	grantee2KeyPrefix := "grantee2"
-	grantee3KeyPrefix := "grantee3"
-	granterKeyPrefix := "default"
-
-	mnemonicAny := genMnemonic(t)
-	gaiaGranterWallet, err := interchaintest.GetAndFundTestUserWithMnemonic(ctx, granterKeyPrefix, mnemonicAny, int64(fundAmount), gaia)
-	require.NoError(t, err)
-
-	mnemonicAny = genMnemonic(t)
-	gaiaGranteeWallet, err := interchaintest.GetAndFundTestUserWithMnemonic(ctx, granteeKeyPrefix, mnemonicAny, int64(granteeFundAmount), gaia)
-	require.NoError(t, err)
-
-	mnemonicAny = genMnemonic(t)
-	gaiaGrantee2Wallet, err := interchaintest.GetAndFundTestUserWithMnemonic(ctx, grantee2KeyPrefix, mnemonicAny, int64(granteeFundAmount), gaia)
-	require.NoError(t, err)
-
-	mnemonicAny = genMnemonic(t)
-	gaiaGrantee3Wallet, err := interchaintest.GetAndFundTestUserWithMnemonic(ctx, grantee3KeyPrefix, mnemonicAny, int64(granteeFundAmount), gaia)
-	require.NoError(t, err)
-
-	mnemonicAny = genMnemonic(t)
-	osmosisUser, err := interchaintest.GetAndFundTestUserWithMnemonic(ctx, "recipient", mnemonicAny, int64(fundAmount), osmosis)
-	require.NoError(t, err)
-
-	mnemonicAny = genMnemonic(t)
-	gaiaUser, err := interchaintest.GetAndFundTestUserWithMnemonic(ctx, "recipient", mnemonicAny, int64(fundAmount), gaia)
-	require.NoError(t, err)
-
-	mnemonic := gaiaGranterWallet.Mnemonic()
-	fmt.Printf("Wallet mnemonic: %s\n", mnemonic)
-
-	rand.Seed(time.Now().UnixNano())
-
-	//IBC chain config is unrelated to RELAYER config so this step is necessary
-	if err := r.RestoreKey(ctx,
-		eRep,
-		gaia.Config(),
-		gaiaGranterWallet.KeyName(),
-		gaiaGranterWallet.Mnemonic(),
-	); err != nil {
-		t.Fatalf("failed to restore granter key to relayer for chain %s: %s", gaia.Config().ChainID, err.Error())
+	var tests = [][]*interchaintest.ChainSpec{
+		{
+			{Name: "gaia", ChainName: "gaia", Version: "v7.0.3", NumValidators: &nv, NumFullNodes: &nf},
+			{Name: "osmosis", ChainName: "osmosis", Version: "v14.0.1", NumValidators: &nv, NumFullNodes: &nf},
+		},
+		{
+			{Name: "gaia", ChainName: "gaia", Version: "v7.0.3", NumValidators: &nv, NumFullNodes: &nf},
+			{Name: "kujira", ChainName: "kujira", Version: "v0.8.7", NumValidators: &nv, NumFullNodes: &nf},
+		},
```

Comment:
The chainspec is hardcoded in the for loop below. 

Maybe I'm missing something here, but I don't think we ever actually use the kujira image. Are we just running the same test twice with a different name?